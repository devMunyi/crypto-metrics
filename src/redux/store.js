import { configureStore, applyMiddleware } from '@reduxjs/toolkit';
import logger from 'redux-logger';
import cryptoReducer from './reducers';

export default configureStore(
  {
    reducer: {
      crypto: cryptoReducer,
    },
  },
  applyMiddleware(logger),
);
