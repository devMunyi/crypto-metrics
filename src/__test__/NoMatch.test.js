import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/store';
import NoMatch from '../pages/NoMatch';

it('renders NoMatch Component correctly', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <NoMatch />
    </Provider>,
  );
  expect(tree).toMatchSnapshot();
});
