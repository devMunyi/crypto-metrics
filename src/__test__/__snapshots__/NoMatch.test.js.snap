// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NoMatch Component correctly 1`] = `
<div>
  NoMatch
</div>
`;
